bin/aplimits

```diff
@@ -265,7 +291,7 @@ class APLimitsMarginalize(APLimits):
         # It's asymmetric because the function is for small numbers.
         # choice of 10, 20 and 100 points is made by looking at functions
```

Comment:
Is this comment out of date now, or is it unrelated to the size of x (i.e. the fact we now use 1000 rather than 100)?

Reply:
Comment updated.